src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestCommand.cs

```diff
@@ -25,17 +24,13 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
         {
             logger.LogDebug($"Wasm Test command called");
 
-            // FIXME This command line parsing should be moved to CLI command arguments
-            //  Parameters: 
-            // - directory name
-            // - test assembly filename
-            // - command line arguments
+            var timeout = _arguments.Timeout.Milliseconds;
+            string engine = _arguments.Engine;
+            string engineArgs = _arguments.EngineArgs;
 
-            var args = new List<string> { "test-assembly-filename" };
-            var cmdline = new CmdLineParser(args.ToArray());
-
-            var runner = new WasmRunner(cmdline.Project);
-            var result = runner.Run();
+            Process runner = Process.Start(engine, engineArgs);
```

Comment:
We have a class called `ProcessManager` available in the Common project that does provide a more safe way to run processes and collects logs. Have a look if you could leverage that

Reply:
Yes, lets try to use the ProcessManager everywhere, will make our life easier, let us know if you need examples.